frontend/src/components/AdminApplication/Presentational.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
-import { IApplicationInfoObject } from '../ApplicationReview';
 import ApplicationOverview from '../ApplicationReview/ApplicationOverview';
+import { IRegisteredApplicationState } from './../Home/reducer';
 
 interface IProps {
-  application: IApplicationInfoObject;
+  application: IRegisteredApplicationState;
 }
 
 const Presentational: React.FunctionComponent<IProps> = (props) => {
   const { application } = props;
 
   return (
     <div className="main-content">
-      <ApplicationOverview applicationInfo={application} title={application.fullname}/>
```

Comment:
Why remove the option to add a title? ApplicationOverview can be used in more places if it's title is not static